### DISCUSSION TITLE
Dose preact has plugin like "eslint-plugin-react-hooks" that could auto fill the hook dependencies?

### DISCUSSION THREAD

**Discussion by chocking** [preactjs/preact#3693] [Q&A] [answered]
Dose preact has plugin like "eslint-plugin-react-hooks" that could auto fill the hook dependencies?
If not, what's the appropriate usage with hook dependencies? Manually check the code and fill the dependencies array?

I found a plugin [eslint-config-preact](https://github.com/preactjs/eslint-config-preact) but I can not let it wokrs.

**Comment by rschristian** [comment]
Doesn't need an alternative. `eslint-plugin-react-hooks` works just fine with Preact; despite the name, it's not dependent on React at all.

Though yes, you should certainly be manually checking the dependencies array and altering when needs be.

> I found a plugin [eslint-config-preact](https://github.com/preactjs/eslint-config-preact) but I can not let it wokrs.

Can you clarify a bit? What doesn't work?

**Comment by chocking** [comment, marked as answer]
Mabe my package.json set type='module', and eslintrc.(c)js not works well with @typescript-eslint/parser, I remove type='module' in my react project and then every thing goes well.
